requirements/test.txt

```diff
@@ -1,3 +1,5 @@
 msprime==1.1.0
 pytest==6.2.5
 pytest-cov==3.0.0
+tskit==0.4.0
+numpy==1.22.1
```

Comment:
I don't think it's necessary to add numpy here too, because its already in the minimal.txt file.